packages/Confirmation/src/Confirmation.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "@paprika/button";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Popover from "@paprika/popover";
+import contentStyles from "./ContentStyles.styles";
+import ConfirmationStyles from "./Confirmation.styles";
+
+const propTypes = {
+  buttonSize: PropTypes.oneOf(ShirtSizes.DEFAULT),
+  confirmButtonType: PropTypes.oneOf(["primary", "destructive"]),
+  confirmLabel: PropTypes.string.isRequired,
+  description: PropTypes.node,
+  isOpen: PropTypes.bool.isRequired,
+  onCancel: PropTypes.func.isRequired,
+  onClose: PropTypes.func,
+  onConfirm: PropTypes.func.isRequired,
+  heading: PropTypes.string.isRequired,
+};
+
+const defaultProps = {
+  buttonSize: ShirtSizes.MEDIUM,
+  confirmButtonType: "destructive",
+  description: null,
+  onClose: () => {},
+};
+
+const Confirmation = props => {
+  const {
+    heading,
+    buttonSize,
+    confirmButtonType,
+    confirmLabel,
+    description,
+    onConfirm,
+    onCancel,
+    onClose,
+    isOpen,
+  } = props;
+  const confirmRef = React.useRef(null);
```

Comment:
could you rename the variable so is clear that the ref is targeting the Button Confirm? 

Without the word button, this seems that this ref would be targeting the entire confirm container.